### ISSUE TITLE
Remove duplicate messsages

### ISSUE THREAD

**Issue by sk-** [sk-/git-lint#29]
Sometimes different linters report the same message and hence is reported twice by the tool.

One such example is in Java with empty catch block

```
line 36: Avoid empty catch blocks
line 36: Avoid empty catch blocks
```
